.env

```diff
@@ -0,0 +1,7 @@
+FIREBASE_PROJECT_ID='fir-course-dca21'
+FIREBASE_APP_ID='1:797180809192:web:d99ab3933165058f789e5d'
+FIREBASE_STORAGE_BUCKET='fir-course-dca21.appspot.com'
+FIREBASE_API_KEY='AIzaSyBZdKx5HMjTckYcLISS3SRbQ-EmFJ47ymY'
+FIREBASE_AUTH_DOMAIN='fir-course-dca21.firebaseapp.com'
+FIREBASE_MESSAGING_SENDER_ID='797180809192'
+FIREBASE_MEASUREMENT_ID='G-QNSKV81H47'
```

Comment:
Typically, an `.env` file shouldn't be included in a git repository because it contains sensitive information that may allow malicious actors to access your database. With Firebase, it is not crucial because the database is secured by the Firestore rules. But excluding this file from your repo is still good practice. Here is an [article](https://www.pluralsight.com/guides/how-to-use-gitignore-file) that explains how to do that.